src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js

```diff
@@ -168,13 +168,15 @@ $(document).ready(function () {
         }
 
         privates.getAllGroupsOptions(function (options) {
-
-            var responseOptions = "<option value = '0'>" + currNs.groupIsNotSelectedMessage + "</option>";
-
-            $.each(options, function (key, value) {
-                responseOptions += "<option value='" + value.Id + "'>" + value.Name + "</option>";
-            });
-       
+            if (options.length == 1) {
+                var responseOptions = "<option value ='" + options[0].Id + "'>" + options[0].Name + "</option>";
+            }
+            else {
+                var responseOptions = "<option value = '0'>" + currNs.groupIsNotSelectedMessage + "</option>";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "responseOptions" as this name is already used in declaration at line 172. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2814)

Reply:
but it is the same field because we use it after conditions

Reply:
https://www.w3schools.com/js/js_hoisting.asp

Reply:
it is my fault, I understood sorry for controversy.